pkgs/by-name/or/orca-slicer/package.nix

```diff
@@ -126,7 +128,11 @@ stdenv.mkDerivation rec {
   patches = [
     # Fix for webkitgtk linking
     ./patches/0001-not-for-upstream-CMakeLists-Link-against-webkit2gtk-.patch
+    # Link opencv_core and opencv_imgproc instead of opencv_world
     ./patches/dont-link-opencv-world-orca.patch
+    # The changeset from https://github.com/SoftFever/OrcaSlicer/pull/7650, can be removed when that PR gets merged
+    # Allows disabling the update nag screen
+    ./patches/pr-7650-configurable-update-check.patch
```

Comment:
Why do we vendor the patch?

Reply:
Mostly thought it was too small to bother fetching it with a web request, and there are no licensing issues or anything like that because I wrote it. Do you think it should be a `fetchurl` or something instead?

Reply:
> Mostly thought it was too small to bother fetching it with a web request

we almost always fetch patches unless they don't cleanly apply or are not applicable to upstream.

> there are no licensing issues

There cannot be any licensing issues here. The is submitted upstream and is under the same license as the software and we properly attribute its source.

---

patches are always fetched using fetchpatch to make sure they do not contain any data that might invalidate the hash later one.